src/main/java/net/minecraftforge/event/LootTableLoadEvent.java

```diff
@@ -38,11 +39,13 @@
 {
     private final ResourceLocation name;
     private LootTable table;
+    private LootTableManager tableManager;
 
-    public LootTableLoadEvent(ResourceLocation name, LootTable table)
+    public LootTableLoadEvent(ResourceLocation name, LootTable table, LootTableManager tableManager)
```

Comment:
Perhaps there is a better way to do this, other then exposing the `LootTableManager` directly?